backend/tests/v5/unit/processors/teamspaces/projects/models/federations.test.js

```diff
@@ -340,15 +340,15 @@ const testGetTicketGroupById = () => {
 			const expectedData = generateRandomObject();
 			const fn = jest.spyOn(TicketGroup, 'getTicketGroupById').mockResolvedValueOnce(expectedData);
 
-			await expect(Federations.getTicketGroupById(teamspace, projectId, federation, revId, ticket, groupId))
+			await expect(Federations.getTicketGroupById(teamspace, projectId, federation, revId, ticket, groupId, true))
```

Comment:
we should probably put true as default on the function definition to avoid unexpected situations